src/bridge/command.rs

```diff
@@ -147,27 +154,25 @@ fn platform_which(bin: &str) -> Option<String> {
 }
 
 #[cfg(target_os = "macos")]
-fn nvim_cmd_impl(bin: &str, args: &[String]) -> TokioCommand {
+fn nvim_cmd_impl(bin: String, mut args: Vec<String>) -> TokioCommand {
     let shell = env::var("SHELL").unwrap_or_else(|_| "/bin/sh".to_string());
+    args.insert(0, bin);
```

Comment:
By inserting the executable path into the head of `args`, I could avoid `format!("{} {}", bin, args)` later. By this, one redundant clone was removed.